algorithm/contour/fromRaster.test.js

```diff
@@ -10,35 +9,35 @@ test('Simple', async (t) => {
     [77, 81, 86, 99],
   ];
   const result = await fromRaster(raster, [0, 25, 50, 75, 100]);
-  t.deepEqual(result.map(canonicalizePaths), [
+  t.deepEqual(result, [
     [
       [
-        [0, 0.57143, 0],
+        [0, 0.5714285714285714, 0],
         [0, 0, 0],
-        [0.57143, 0, 0],
+        [0.5714285714285714, 0, 0],
       ],
     ],
     [
       [
-        [0.57143, 0, 25],
+        [0.5714285714285714, 0, 25],
         [1, 0, 25],
         [2, 0, 25],
         [3, 0, 25],
-        [2.77778, 1, 25],
-        [2.26667, 2, 25],
-        [2, 2.26667, 25],
-        [1, 2.64706, 25],
-        [0, 2.88889, 25],
+        [2.7777777777777777, 1, 25],
```

Comment:
Why the rounding change?

Reply:
I've been removing canonicalize.

It may want replacing, but let's see.